src/checkbox/Checkbox.ts

```diff
@@ -21,6 +21,7 @@ import * as css from '../theme/checkbox/checkbox.m.css';
  */
 export interface CheckboxProperties extends ThemedProperties, InputProperties, LabeledProperties, InputEventProperties, KeyEventProperties, PointerEventProperties, CustomAriaProperties {
 	checked?: boolean;
+	id?: string;
```

Comment:
Could this be added to `InputProperties` or other type in the `interfaces` file instead? Otherwise we're back to repeated properties.

Reply:
I'm good with it in `InputProperties` if you are 👍. I left it out initially because it didn't seem an input-specific thing, but we can say they're being added for input-related reasons.